Snippets/Snippets_3/DelayedDelivery/DeferForDateTime.cs

```diff
@@ -0,0 +1,26 @@
+namespace Snippets3.DelayedDelivery
+{
+    using System;
+    using NServiceBus;
+
+    class DeferForDateTime
+    {
+        IBus bus;
+
+        public DeferForDateTime(IBus bus)
+        {
+            this.bus = bus;
```

Comment:
no point injecting this if it is not runnable code. just do a null local variable